spec/requests/workflows/destroy_all_spec.rb

```diff
@@ -11,11 +11,12 @@
     let!(:item2) do
       FactoryBot.create(:workflow_step,
                         repository: 'sdr',
+                        process: 'publish',
```

Comment:
necessary for tests to continue to work with our new validation to prevent dupe steps from being added